src/benchmarks/micro/libraries/System.Runtime.Numerics/Perf.BigInteger.cs

```diff
@@ -106,6 +106,44 @@ public IEnumerable<object> ModPowValues()
         public BigInteger ModPow(BigIntegers arguments)
             => BigInteger.ModPow(arguments.Left, arguments.Right, arguments.Other);
 
+        public IEnumerable<object> EqualsValues()
+        {
+            Random rnd = new Random(123456);
+
+            foreach (int byteCount in new[] { 5, 19, 67, 259 })
+            {
+                byte[] bytes = new byte[byteCount];
+
+                do
+                {
+                    rnd.NextBytes(bytes);
+                } while (bytes[^1] is not (> 0 and < 128));
+
+                BigInteger b1 = new(bytes);
+                yield return new BigIntegers(b1, new BigInteger(bytes), $"{byteCount} bytes, Same");
+
+                byte copy = bytes[^1];
+                bytes[^1] = (byte)(~bytes[^1] & 0x7F);
+                yield return new BigIntegers(b1, new BigInteger(bytes), $"{byteCount} bytes, DiffLastByte");
+                bytes[^1] = copy;
```

Comment:
I love the new C# syntax, but this repo needs to compile against older SDKs and monikers so we can detect regressions between releases.

```suggestion
                byte[] bytes = new byte[byteCount];
                int lastByte = bytes.Length - 1;

                do
                {
                    rnd.NextBytes(bytes);
                } while (bytes[lastByte] is not (> 0 and < 128));

                BigInteger b1 = new(bytes);
                yield return new BigIntegers(b1, new BigInteger(bytes), $"{byteCount} bytes, Same");

                byte copy = bytes[lastByte];
                bytes[lastByte] = (byte)(~bytes[lastByte] & 0x7F);
                yield return new BigIntegers(b1, new BigInteger(bytes), $"{byteCount} bytes, DiffLastByte");
                bytes[lastByte] = copy;
```